abci/types/application.go

```diff
@@ -98,7 +98,7 @@ func (BaseApplication) ApplySnapshotChunk(req RequestApplySnapshotChunk) Respons
 }
 
 func (BaseApplication) PreprocessTxs(req RequestPreprocessTxs) ResponsePreprocessTxs {
-	return ResponsePreprocessTxs{}
+	return ResponsePreprocessTxs{Txs: req.Txs}
```

Comment:
this took entirely too long to debug... not that it matters too much cause upstream removed these tests in [#6685](https://github.com/tendermint/tendermint/pull/6684)